pkg/lib/bundle/supported_resources.go

```diff
@@ -16,6 +16,7 @@ const (
 	PodDisruptionBudgetKind   = "PodDisruptionBudget"
 	PriorityClassKind         = "PriorityClass"
 	VerticalPodAutoscalerKind = "VerticalPodAutoscaler"
+	ConsoleYamlSampleKind     = "ConsoleYamlSample"
```

Comment:
I think these need to be added to OLM before we allow them to be added to bundles: https://github.com/operator-framework/operator-lifecycle-manager/blob/78bb41b4613cafca7264a3108d1f1fa4b66024c8/pkg/controller/operators/catalog/operator.go#L2080

Reply:
Ah, I see there's a PR open for it: https://github.com/operator-framework/operator-lifecycle-manager/pull/1616